include/gis.h

```diff
@@ -60,7 +60,7 @@ static const char *GRASS_copyright __attribute__ ((unused))
 #define FALSE 0
 #endif
 
-#if defined(_FILE_OFFSET_BITS) && _FILE_OFFSET_BITS == 64
+#if SIZEOF_OFF_T == SIZEOF_LONG_LONG_INT
```

Comment:
What is wrong with `_FILE_OFFSET_BITS`?
`SIZEOF_*` is not safe when cross-compiling, e.g. with mingw.

Reply:
_FILE_OFFSET_BITS is not defined for clang, at least on Mac (don't know for other platforms). I wrongly interpreted
 the readings of CI build logs for linux, believing this check fails also there. But this seems to be Mac only.
Updated the check accordingly.